sphinx/conf.py

```diff
@@ -41,15 +62,10 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.)
-extensions = ['sphinx.ext.extlinks', 'edit_on_github']
```

Comment:
This extensions is no longer necessary as it is built into readthedocs?

Reply:
That should have been removed

Reply:
actually now this is required for ``mvn`` build if one is building locally